x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/edit_policy.test.ts

```diff
@@ -92,5 +106,31 @@ describe('<EditPolicy />', () => {
       const latestRequest = server.requests[server.requests.length - 1];
       expect(JSON.parse(JSON.parse(latestRequest.requestBody).body)).toEqual(expected);
     });
+
+    test('wait for snapshot field should display a callout when there are no snapshot policies', async () => {
+      httpRequestsMockHelpers.setLoadSnapshotPolicies([]);
+
+      await act(async () => {
+        testBed = await setup();
```

Comment:
nit; might be worth adding a comment here about why we are calling setup again (outside of the beforeEach). WDYT?

Reply:
yes, I was debating this structure anyways! I think a comment will help here